.circleci/config.yml

```diff
@@ -213,8 +220,8 @@ jobs:
       - restore_cache:
           name: Restore Yarn Cache
           keys:
-            - yarn-deps-{{ checksum "ui/yarn.lock" }}
-            - yarn-deps-
+            - yarn-deps-lock-{{ checksum "ui/yarn.lock" }}
+            - yarn-deps-lock-
```

Comment:
Changing the `WORKDIR` in the CI image somehow broke permissions for `/go`, so all existing caches containing `/go/pkg/mod` trigger permissions errors on extraction. Have to bust the cache.